git-verify-tag: add page (#6139)
tldr-pages/tldr: pages/common/git-verify-tag.md

# git verify-tag

> Check for GPG verification of tags.

> > If tag isn't signed with -s when made, you will get an error regardless of signing.

Reviewer: How about a shorter version:
```suggestion
> If a tag wasn't signed, an error will occur.
```

> > If tag isn't signed with -s when made, you will get an error regardless of signing.

Reviewer: How about a shorter version:
```suggestion
> If a tag wasn't signed, an error will occur.
```

Author: signed manually would be better since it seems that regardless of it being signed, the -s flag seems to flag it up for git, almost as if it is treated different

Reviewer 2: `with -s` without the full command doesn't have enough sense, also I think should be a `gitconfig` setting to auto sign tags.

Reviewer 3: I prefer @marchersimon's suggestion — less complexity means less likelyhood of misunderstanding the conversation.

Author: @navarroaxel it does sign the tags with `	gpgsign = true` but not flagged for some reason

Author: ```suggestion
> If a tag wasn't manually signed, an error will occur.
```

Reviewer 2: @CleanMachine1 `manually` or `automatic` is not important here, @marchersimon suggestion is good enough.

Author: I disagree but others agree, so OK

> More information: <https://git-scm.com/docs/git-verify-tag>.

- Check tags for a GPG signature:

`git verify-tag {{tag1 optional_tag2 ...}}`

> - Check tags for a GPG signature and show details of each tag:

Reviewer: ```suggestion
- Check tags for a GPG signature and show details for each tag:
```
